legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -254,6 +254,22 @@ class FilingTitles(str, Enum):
             {'types': 'BC,BEN,CC,ULC', 'outputs': ['letterOfConsent', ]},
         ]
     },
+    'continuationIn': {
+        'name': 'continuationIn',
+        'title': 'Continuation In',
+        'displayName': {
+            'BC': 'BC Limited Company Continuation In Application',
+            'BEN': 'BC Benefit Company Continuation In Application',
+            'CC': 'BC Community Contribution Company Continuation In Application',
+            'ULC': 'BC Unlimited Liability Company Continuation In Application',
+        },
+        'codes': {
+            'BC': 'CONTI',
+            'BEN': 'CONTI',
+            'ULC': 'CONTI',
+            'CC': 'CONTI'
```

Comment:
Are these the fee codes? Does this cover off bcgov/entity#19284?

Reply:
I referenced to ticket https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/19283

Reply:
Can 19284 be closed then?

Reply:
Yes. I think so. I can add reference to this PR